src/PrestaShopBundle/Bridge/AdminController/FrameworkBridgeControllerListTrait.php

```diff
@@ -28,28 +28,35 @@
 
 namespace PrestaShopBundle\Bridge\AdminController;
 
-use PrestaShopBundle\Bridge\AdminController\Action\ActionInterface;
-use PrestaShopBundle\Bridge\Exception\NotAllowedActionTypeForListException;
-use PrestaShopBundle\Bridge\Helper\Listing\Action\ListBulkAction;
-use PrestaShopBundle\Bridge\Helper\Listing\Action\ListHeaderToolbarAction;
-use PrestaShopBundle\Bridge\Helper\Listing\Action\ListRowAction;
-use PrestaShopBundle\Bridge\Helper\Listing\Field\FieldInterface;
-use PrestaShopBundle\Bridge\Helper\Listing\FiltersProcessor;
 use PrestaShopBundle\Bridge\Helper\Listing\HelperListConfiguration;
+use Symfony\Component\HttpFoundation\Request;
 
 /**
  * Contains the principal methods you need to horizontally migrate a controller which has a list.
  */
 trait FrameworkBridgeControllerListTrait
 {
+    /**
+     * @param string $identifierKey
+     * @param string $positionIdentifierKey
+     * @param string $defaultOrderBy
+     * @param bool $autoJoinLangTable
+     * @param bool $deleted
+     * @param bool $explicitSelect
+     * @param bool $useFoundRows
+     * @param string|null $listId
+     *
+     * @return HelperListConfiguration
+     */
     protected function buildListConfiguration(
         string $identifierKey,
         string $positionIdentifierKey,
         string $defaultOrderBy,
         bool $autoJoinLangTable = true,
         bool $deleted = false,
         bool $explicitSelect = false,
-        bool $useFoundRows = true
+        bool $useFoundRows = true,
+        ?string $listId = null
```

Comment:
looking at leagacy it seems to be used as an optional param - in case someone wants to pass different list id than tableName. So when building the configuration its not mandatory as 90% of the case it will probably be the same as table name. So i think its just convenient to have it as optional param :thinking:

Reply:
OK, makes sense